### ISSUE TITLE
Names of array of variables

### ISSUE THREAD

**Issue by ilarioc** [JuliaDiffEq/ModelingToolkit.jl#247]
Hi,
I noticed that variable with indexes have a symbol between numbers
```julia
Variable("x", 1, 2, 3)
```
prints
```
x₁̒₂̒₃
``` 
Is this on purpose?

**Comment by ChrisRackauckas** [comment]
Yes, and I know it's not the best, but it's the unicode we have for now. Do you have a better suggestion for a unicode separator between the numbers? Because we want things like `x_{11,2}`, but I couldn't find a subscript , so I put in what I could find for now :shrug:. It's easy to change so if you find a better unicode we can swap it out.

------------------------------------------------
Applied Mathematics Instructor, Massachusetts Institute of Technology
Senior Research Analyst, University of Maryland, Baltimore, School of Pharmacy, Center for Translational Medicine
On 2/19/2020 12:37:00 PM, ilarioc <notifications@github.com> wrote:
Hi,
I noticed that variable with indexes have a symbol between numbers
Variable("x", 1, 2, 3)
prints
x₁̒₂̒₃
Is this on purpose?

**Comment by ilarioc** [comment]
I hadn't thought about the case of multiple digits and was wondering if one could do without it. An alternative could be to use ˏ (U+02CF) which looks more like a comma ```x₁₁ˏ₂```. It's not ideal but seems to be rendered more consistently across different shells (the current one gets mixed with the next number on my system).

**Comment by ChrisRackauckas** [comment]
yeah that's certainly better.

**Comment by ChrisRackauckas** [comment]
\u2e12 was also suggested, but `u₁⸒₂` isn't a valid variable name.

**Comment by ChrisRackauckas** [comment]
It's been changed and improved, but leaving open for further discussion and possibly a future change.
